cmd/demo/node.go

```diff
@@ -122,6 +126,17 @@ func importAccount(secret string) (*ewallet.Wallet, *ewallet.Account, error) {
 	return wallet, acc.(*ewallet.Account), err
 }
 
+func getOnChainBal(addressList []wallet.Address) (balList []*big.Int, err error) {
+	for _, addr := range addressList {
+		bal, err := ethereumBackend.BalanceAt(context.Background(), common.BytesToAddress(addr.Bytes()), nil)
```

Comment:
We should put at least some timeout here, otherwise a disconnected `eth-node` will block indefinitely.

Reply:
`ewallet.AsEthAddr(addr)` instead of the `BytesToAddress` should work.

Reply:
Yep, never do `context.Background()` calls. Cleanest solution would be to give `getOnChainBal` a context as first parameter. Then the caller can pass a sensible context.

I also propose @ggwpez @manoranjith that we rename the `Config.Chain.DeployTimeout` to `TxTimeout` so we have a configurable transaction timeout for txs that don't include any other timeouts (like challenge durations). Then we can use this timeout for calls to `getOnChainBal` if no other context is available.

Reply:
Added a context argument to getOnChainBal and passed a context.WithTimeout(DeployTimeout) when the function is called. See this commit - 85c0766. Should we rename the DeployTimeout ? @ggwpez

Reply:
Yes please also rename (as separate commit) the field to `TxTimeout`. Make sure that the field name is also fixed in the configuration files. @ggwpez knows best other effects of this change.

Reply:
That would create changes in `config.go`, `alice.yaml`, `bob.yaml` and at two spots in `node.go` where it is used.

Reply:
Addressed in this commit - 8f2e300.